x-pack/legacy/plugins/siem/public/components/drag_and_drop/drag_drop_context_wrapper.tsx

```diff
@@ -57,43 +57,42 @@ const onDragEndHandler = ({
 /**
  * DragDropContextWrapperComponent handles all drag end events
  */
-export class DragDropContextWrapperComponent extends React.Component<Props> {
-  public shouldComponentUpdate = ({ children, dataProviders }: Props) =>
-    children === this.props.children && dataProviders !== this.props.dataProviders // prevent re-renders when data providers are added or removed, but all other props are the same
-      ? false
-      : true;
-
-  public render() {
-    const { children } = this.props;
+export const DragDropContextWrapperComponent = React.memo<Props>(
+  ({ browserFields, children, dataProviders, dispatch }) => {
+    const onDragEnd: (result: DropResult, provided: ResponderProvided) => void = (
```

Comment:
instead of having this lambda function `onDragEnd` to rebuild each time the component render, can we do it with a function like that
```
function onDragEnd(result: DropResult) {
      enableScrolling();

      if (dataProviders != null) {
        onDragEndHandler({
          browserFields,
          result,
          dataProviders,
          dispatch,
        });
      }

      if (!draggableIsField(result)) {
        document.body.classList.remove(IS_DRAGGING_CLASS_NAME);
      }
    }
```